src/OperatorMsg.chpl

```diff
@@ -148,7 +148,9 @@ module OperatorMsg
             return doBinOpvv(l, r, e, op, rname, pn, st);
           }
         }
-        return new MsgTuple("Bin op not supported", MsgType.NORMAL);
+        var errorMsg = notImplementedError(pn,left.dtype,op,right.dtype);
```

Comment:
https://github.com/Bears-R-Us/arkouda/blob/ce3c947ef03750e62035cb326a8d75b7e32eb5fe/src/OperatorMsg.chpl#L585-L590

Reply:
whoops, also, the MsgType should be MsgType.ERROR